### ISSUE TITLE
Daily Average Overview

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [alexjustesen/speedtest-tracker#1541]
Check this out:

![image](https://github.com/alexjustesen/speedtest-tracker/assets/3938423/93af30d4-4a47-46a3-b8fd-f263e9dfa3f1)

It's from https://github.com/henrywhitaker3/Speedtest-Tracker . Would be nice if this speed-tracker had something like that too. Just the Average from the last 24 hours to post on Telegram channel or via mail.

**Comment by coro1404** [comment]
As my connection is highly volatile (T-DSL Hybrid, DSL+LTE aggreated with QOS for shared medium LTE) it would be cool to see the average also in the daily / weekly / monthly  graphs - because in the end I'm only interested if I get about what I paid vor (e.g. >= 100 MBit/s on average) - adding as feature or enhancemend request here.

**Comment by jwaugh** [comment]
> As my connection is highly volatile (T-DSL Hybrid, DSL+LTE aggreated with QOS for shared medium LTE) it would be cool to see the average also in the daily / weekly / monthly graphs - because in the end I'm only interested if I get about what I paid vor (e.g. >= 100 MBit/s on average) - adding as feature or enhancemend request here.

I agree! Henry Whitaker's averages were nice to have when I used that container. It just got too clunky for me and found that Alex's is easier to work with. I'm very happy he implemented the averages line on the graphs. That's a little better. Would still love to see Henry Whitaker's version of "text stats" be implemented. It was handy.

**Comment by alexjustesen** [comment]
With the newly implemented API, would a stats endpoint be useful to also grab these data points in a non-ui manner?

**Comment by Tiritibambix** [comment]
> Would still love to see Henry Whitaker's version of "text stats" be implemented. It was handy.

I do agree
